src/store/modules/wallet.js

```diff
@@ -105,7 +106,7 @@ const mutations = {
       state.email = '';
       state.emailUnverified = '';
       state.followees = [];
-      state.eventLastSeenTs = 0;
+      state.eventLastSeenTs = 1;
```

Comment:
Why 1 instead of 0?

Reply:
since `getHasUnseenEvents: state =>
    state.eventLastSeenTs && ....`  may return false if eventLastSeenTs = 0

Reply:
How about -1